Gateway/Command/CreatePayment.php

```diff
@@ -53,6 +53,10 @@ public function execute(array $commandSubject)
             ]
         ];
 
+        if ($type = $payment->getMethodInstance()->getCaptureType()) {
+            $data['input']['captureType'] = $type;
```

Comment:
```suggestion
        if ($captureType = $payment->getMethodInstance()->getCaptureType()) {
            $data['input']['captureType'] = $captureType;
```
as $type is already used in a different context.

Reply:
Resolved